fluentd/run.sh

```diff
@@ -14,29 +14,74 @@ docker_uses_journal() {
     grep -q "^OPTIONS='[^']*--log-driver=journald" /etc/sysconfig/docker
 }
 
-if [ "${USE_JOURNAL:-}" != false ] ; then
-    if [ -z "${JOURNAL_SOURCE:-}" ] ; then
-        if [ -d /var/log/journal ] ; then
-            export JOURNAL_SOURCE=/var/log/journal
-        else
-            export JOURNAL_SOURCE=/run/log/journal
+if [ "${MUX_ALLOW_EXTERNAL:-}" = "true" ] ; then
+    # mux service implies mux
+    export USE_MUX=true
+fi
+
+if [ "${USE_MUX:-}" != "true" ] ; then
+    if [ "${USE_JOURNAL:-}" != false ] ; then
+        if [ -z "${JOURNAL_SOURCE:-}" ] ; then
+            if [ -d /var/log/journal ] ; then
```

Comment:
@t0ffel no - but this PR does not change that behavior - please open a separate issue, or suggest a way that fluentd can know which directory to listen to, or suggest a way that fluent can listen to both /var/log/journal and /run/log/journal from the host.